### ISSUE TITLE
[CHANGE] Tiebreaker rules for the scoreboard

### ISSUE THREAD

**Issue by rodylg** [odamex/odamex#880]
**What is the existing behavior? Please describe.**
Scoreboard seems to break ties on the table by using "min".

**What is the desired behavior? Why are you requesting the change?**
Announcing the game ending on a draw on tied frags is good, but on the scoreboard ties maybe should break first by deaths (fewer deaths should be rewarded) and then by min. If I understand correctly, "min" is the minimum time the player stayed alive and such metric, although it may be indicative of skill, is more related to respawning luck (and if on-spawn telefrags count that's almost unfair). Fewer deaths on the same frag count lead to a higher K/D ratio, which is more indicative of skill.

**If applicable, describe alternatives you've considered**
Breaking ties using K/D ratio is an alternative, although since the players are tied in kills the resulting behaviour would be the same as using fewer deaths (but the coding might be less straightforward).

**Additional context**
IMO JSharp should have been first.
![Scoreboard tie](https://github.com/odamex/odamex/assets/49259699/e7191879-15b0-4163-bbfb-af151434068d)

**Comment by rakohus** [comment]
MIN is just total amount of time in server, in minutes.

**Comment by electricbrass** [comment]
I don't believe any sorting is done beside that based on the frags. Players with equal frags are just left in whatever order the game had them stored in.

**Comment by rodylg** [comment]
I took a quick dive into the code and I think that's true since it's only using a single std::sort call to order the player list. Maybe, would it be better if this ticket was converted into a feature request?

**Comment by rakohus** [comment]
Change is probably fine as it's a change to existing scoreboard functionality
